### PR TITLE
fix for feature2d java wrappers as described in this post: http://ans…

### PR DESCRIPTION
**Pull request by abratchik**
http://answers.opencv.org/question/101675/surfsift-java-wrapper-for-opencv-3xosx-1011/

### CODE REVIEW

**Comment by vpisarev** [comment]
@mshabunin, could you please review it?

File: modules/java/generator/gen_java.py [hunk 1]
```diff
@@ -1347,7 +1349,7 @@ def gen_func(self, ci, fi, prop_name=''):
                 ret = "return (jlong) new %s(_retval_);" % self.fullTypeName(fi.ctype)
             elif fi.ctype.startswith('Ptr_'):
                 c_prologue.append("typedef Ptr<%s> %s;" % (self.fullTypeName(fi.ctype[4:]), fi.ctype))
-                ret = "%(ctype)s* curval = new %(ctype)s(_retval_);return (jlong)curval->get();" % { 'ctype':fi.ctype }
+                ret = "%(ctype)s* curval = new %(ctype)s(_retval_);return (jlong)curval;" % { 'ctype':fi.ctype }
```
Comment (mshabunin) [inline modules/java/generator/gen_java.py]: Please, revert this line, same change has been integrated already.

**Comment by mshabunin** [comment]
:+1:

**Comment by mshabunin** [comment]
@abratchik , I guess you need to create additional pull request to _opencv_contrib_ in case to enable Java wrappers for _xfeatures2d_ module.

**Pull request merged**